.github/workflows/CI.yml

```
@@ -31,31 +33,25 @@ jobs:
     strategy:
       matrix:
         os: [ubuntu-latest, windows-latest, macOS-latest]
-        eslint: [8.x]
+        eslint: [8.x, "9.0.0-alpha.0"]
         node: [18.x, 20.x]
         include:
```

It may be worth keeping the minimim specs here (mostly because I keep accidentally using unsupoported things :sweat_smile:)

```suggestion
        include:
          - os: ubuntu-latest
            node: "18.8.0"
            eslint: "8.x"
          - os: ubuntu-latest
            node: "20.9.0"
            eslint: "8.x"
          - os: ubuntu-latest
            node: "21.1.0"
            eslint: "8.x"
```